# **PYTHON PROJECT : CREATE MATHEMATICAL TABLE OF ANY NUMBER**
---
- ## Write a program that takes an integer as input, and creates a mathematical table of that number.
- ## Example :
### 1 x number = result
### 2 x number = result
### 3x number = result
### .
### .
### .
### 10 x number = result
---
## Defining a function 'multiplication()' which will take an input 'num'.
- ### Here the logic is - we want to multiply the parameter 'num' by each number from 1 to 10.

In [ ]:
def multiplication(num) :
  for i in range(1,11) :  # Assigning numbers one by one to 'i' from 1 to 10 using for loop
    result = i*num
    print(i,'x',num,'=',result)  # multiplying each number from 1 to 10 with the input number and print the result

num = int(input("Enter a number : ")) # Taking input from user
multiplication(num)  # Calling the function we have defined

Enter a number : 49
1 x 49 = 49
2 x 49 = 98
3 x 49 = 147
4 x 49 = 196
5 x 49 = 245
6 x 49 = 294
7 x 49 = 343
8 x 49 = 392
9 x 49 = 441
10 x 49 = 490


### So in the output we get multiplication table of 49 all the way upto 490.